README.md

```diff
@@ -141,15 +142,130 @@ for (index, game) in myFavoriteZeldaGames.enumerate() {
     print("\(index + 1). The Legend of Zelda: \(game)")
 }
 
-// 1. The Legend of Zelda: Ocarina of Time
-// 2. The Legend of Zelda: A Link to the Past
-// 3. The Legend of Zelda: Majora's Mask
-// 4. The Legend of Zelda: Wind Waker
-// 5. The Legend of Zelda: Twighlight Princess
+// 1\. The Legend of Zelda: Ocarina of Time
+// 2\. The Legend of Zelda: A Link to the Past
+// 3\. The Legend of Zelda: Majora's Mask
+// 4\. The Legend of Zelda: Wind Waker
+// 5\. The Legend of Zelda: Twighlight Princess
+```
+
+- If we do cover enumerate, maybe we VERY briefly talk about Tuples (but not too much because Tuples will be brought up in another unit.
+
+--------------------------------------------------------------------------------
+
+## Improving Repetitive Tasks
+
+In the last lesson you saw how to create and manipulate a collection of values. You also saw how to print individual items in that collection, so if you wanted to print a list of your three friends you could do the following:
+
+```swift
+let friends = ["Chris", "Billy", "Jessica"]
+print (friends[0])
+// "Chris"
+print (friends[1])
+// "Billy"
+print (friends[2])
+// "Jessica"
+```
+
+But what if you had 100 friends? Printing values like this will become tedious quickly and if you added more friends to the list, you would have to add more `print` commands to handle them.
+
+Programming languages are very good at repetitive tasks, in fact they are better than humans, so in this lesson you will learn how to perform the same action(s) on values in an array multiple times. This is called _iterating over an array_ and introduces a new concept, a _loop_.
+
+## Loops
+
+```swift
+let friends = ["Chris", "Billy", "Jessica"]
+
+for friend in friends {
+    print("Hello \(friend)")
+}
+
+// Hello Chris
+// Hello Billy
+// Hello Jessica
+```
+
+This examples starts with a familiar array initialization, but then instead of using the `print` command for each value in the array, says that `for` each value in the `friends` array, process the instructions in-between the `{}` symbols, which are called 'curly brackets' or 'mustaches'. In this example these instructions print the current value of `friend`, or the value of the current index in the `friends` array.
+
```

Comment:
This is really good. One thing I think we can add to this is letting the student know the order these items will print.. or something like that.

for in loops iterate starting from index 0 (first item in the array) to the last item in the array in order every time. It will never be random.